src/main/resources/templates/final.html

```diff
@@ -6,7 +6,7 @@
 </head>
 <body>
     <div th:replace="header :: header">...</div>
-    <div class="container" style="padding-top: 50px;"> this is final view</div>
+    <div class="container ocdu-container"> this is final view</div>
```

Comment:
since this code is repeating it would be better to make a subtemplate out of it and use thymeleaf directive to insert it here.